### PR TITLE
Clean Rust and C docs

### PR DESCRIPTION
**Pull request by SilasMarvin**

### CODE REVIEW

File: pgml-cms/docs/api/client-sdk/README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ async fn main() -> Result<(), Error> {
 {% endtab %}
 
 {% tab title="C" %}
-```c
+```cpp
```
Comment (levkk) [inline pgml-cms/docs/api/client-sdk/README.md:102]: You may need to install the cpp syntax highligher extension for our text editor.

Comment (SilasMarvin) [reply pgml-cms/docs/api/client-sdk/README.md:102]: I did!

Comment (SilasMarvin) [reply pgml-cms/docs/api/client-sdk/README.md:102]: There was two prs, this was meant to be in the first one but I accidentally merged it, so had to add this one after.

https://github.com/postgresml/postgresml/pull/1506

Comment (levkk) [reply pgml-cms/docs/api/client-sdk/README.md:102]: Nice!

**Pull request merged**